test/language/expressions/assignmenttargettype/direct-primaryexpression-objectliteral.js

```diff
@@ -1,19 +0,0 @@
-// This file was procedurally generated from the following sources:
-// - src/assignment-target-type/primaryexpression-objectliteral.case
-// - src/assignment-target-type/invalid/direct.template
-/*---
-description: PrimaryExpression ObjectLiteral; Return invalid. (Direct assignment)
-flags: [generated]
-negative:
-  phase: parse
-  type: SyntaxError
-info: |
-    Direct assignment
----*/
-
-$DONOTEVALUATE();
-
-function _() {
-  {} = 1;
```

Comment:
differently from what this PR points out, there is no specified qualification if this is interpreted as a Block and an invalid Syntax `= 1`. Actually, the spec qualifies some initial grammar for `{} = 1;` that is then prohibited only by the static error pointed out in this test. This AssignmentExpression is invalid because the LHS Expression is an invalid AssignmentTargetType. Look at what folder this test is?